.github/workflows/test-jvm.yml

```diff
@@ -12,6 +12,15 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '17'
+      - name: Cache gradle
+        uses: actions/cache@v4
+        env:
+          cache-name: cache-gradle
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
```

Comment:
If we're caching the wrapper, we'll need to add `gradle/wrapper` (in this repository) to the hash input too, right? So it works correctly when we're upgrade the gradle wrapper version?!